src/components/object_store/src/obkv/meta.rs

```diff
@@ -113,13 +116,9 @@ pub struct ObkvObjectMeta {
     /// table_name @ path @ upload_id
     #[serde(rename = "unique_id")]
     pub unique_id: Option<String>,
-    /// The size in bytes of one part. Note: maybe the size of last part less
-    /// than part_size.
-    #[serde(rename = "part_size")]
-    pub part_size: usize,
-    /// The paths of multi upload parts.
+    /// The paths and size of multi upload parts.
     #[serde(rename = "parts")]
-    pub parts: Vec<String>,
+    pub parts: Vec<(String, usize)>,
```

Comment:
This struct is serialized, so it's breaking changes to update its fields?

If it's, we can define a proto to serialize this struct instead of using serde.